api/__init__.py

```diff
@@ -8,3 +9,4 @@
 )
 
 api.add_namespace(fund_ns)
+api.add_namespace(search_ns)
```

Comment:
Namespaces are great for keeping our code modular and tidy!

Reply:
Nice precedent set